composer.json

```diff
@@ -33,12 +33,12 @@
     },
     "require": {
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-servicemanager": "^3.14.0",
```

Comment:
IMO OK to introduce this, since this package ships with symbols that depend on it, but kinda sad to see that the dependency set increases :|